src/components/blog-creator/BlogCreatorHeading.tsx

```diff
@@ -44,13 +44,18 @@ const Heading = styled.header`
   }
 `
 
+const JoinUsLink = styled(ExternalLink)`
+  ${BtnLink}
+`
+
+
 const ConnectedSubmitButton = () => {
-  const ctx = useJoinUsModal()
+  const discordUrl = getDiscordUrl()
 
   return (
-    <Button className="submit-article-btn" onClick={ctx.open}>
+    <JoinUsLink href={discordUrl} id="blog-join-us">
```

Comment:
**I have here idea**

getDiscordUrl() repeats logic which is already done inside `gatsby-node.js`. We're loading all data connected with pages once, to avoid query repetition and because it takes less time than multiple queries.

To avoid `prop-drilling` here, you can use render-slot pattern from React. So to `EmptyAuthorTileProps` you can add property -> submitTrigger which will be type of `ReactNode`, so it allow us to pass any trigger button without messing this component API.